config/routes.rb

```diff
@@ -16,7 +16,12 @@
 
   resources :items
 
-  resources :admins
+  namespace :admin do
+    get '/dashboard', to: 'base#dashboard'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.